armi/materials/material.py

```diff
@@ -108,6 +121,12 @@ def name(self):
         .. impl:: The name of a material is accessible.
             :id: I_ARMI_MAT_NAME
             :implements: R_ARMI_MAT_NAME
+
+            Every instance of an ARMI material must have a simple, human-readable
+            string name. And, if possible, we want this string to match the class
+            name. (This, of course, puts some limits on both the string and the
+            class name.) These names must easily be retrievable as a class
+            property.
```

Comment:
```suggestion
            Every instance of an ARMI material must have a simple, human-readable
            string name. And, if possible, we want this string to match the class
            name. (This, of course, puts some limits on both the string and the
            class name.) These names are easily retrievable as a class property.
```
Not sure if the impl description should say something like "These names are easily retrievable as a class property, **as shown here**." since the intent is for it to go into the documentation. That's where my mind wanted to go with the text, feel free to add if you think that is good or ignore if you don't.